modules/friendlytalkback.js

```diff
@@ -360,53 +380,13 @@ var callback_evaluate = function(e) {
 
 	var text;
```

Comment:
```suggestion
	var text = '\n\n';
```

Common to all notices.

Reply:
👍